faq.html

```diff
@@ -0,0 +1,167 @@
+---
+layout: default
+title: FAQ
+---
+
+<div class="faq container">
+    <h2 class="mt-5 mb-5">FAQ</h2>
+    <section class="container my-5">
+        <section id="accordion">
+            <!--FAQ 1-->
+            <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="faq-1" aria-expanded="false" data-toggle="collapse" href="#faq-1" role="button">
+                <div class="position-relative d-flex align-items-left">
+                    <h2 class="h4 m-0 pr-3">Should I apply for the mentorship?</h2>
+                    <div class="position-absolute top-0 right-0 h-100 d-flex align-items-center">
+                        <i class="fa fa-plus"></i>
+                    </div>
+                </div>
+            </a>
+            <div class="collapse" id="faq-1">
+                <div class="card card-body border-0 p-0">
+                    <p>Women Who Code London mentorship programme is for....</p>
+                </div>
+            </div>
+
+            <!--FAQ 2-->
+            <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="faq-2" aria-expanded="false" data-toggle="collapse" href="#faq-2" role="button">
+                <div class="position-relative d-flex align-items-left">
+                    <h2 class="h4 m-0 pr-3">What type of mentoring is available?</h2>
+                    <div class="position-absolute top-0 right-0 h-100 d-flex align-items-center">
+                        <i class="fa fa-plus"></i>
+                    </div>
+                </div>
+            </a>
+            <div class="collapse" id="faq-2">
+                <div class="card card-body border-0 p-0">
+                    <p>Ad hoc</p>
+                    <p>Long-term</p>
+                </div>
+            </div>
+
+            <!--FAQ 3-->
+            <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="faq-3" aria-expanded="false" data-toggle="collapse" href="#faq-3" role="button">
+                <div class="position-relative d-flex align-items-left">
+                    <h2 class="h4 m-0 pr-3">What does the programme schedule look like?</h2>
+                    <div class="position-absolute top-0 right-0 h-100 d-flex align-items-center">
+                        <i class="fa fa-plus"></i>
+                    </div>
+                </div>
+            </a>
+            <div class="collapse" id="faq-3">
+                <div class="card card-body border-0 p-0">
+                    <p>The timeline.....</p>
+                    <p></p>
+                </div>
+            </div>
+
+            <!--FAQ 4-->
+            <a class="py-3 d-block h-100 w-100 position-relative z-index-1 pr-1 text-secondary border-top" aria-controls="faq-4" aria-expanded="false" data-toggle="collapse" href="#faq-4" role="button">
```

Comment:
```suggestion
            <a class="question" aria-controls="faq-4" aria-expanded="false" data-toggle="collapse" href="#faq-4" role="button">
       
```

and in `faq.scss` you can create a class: 

```
.question {
@extend .ppy-3;
@extend .d-block;
@extend .h-100;
@extend .w-100;
@extend .position-relative;
@extend .z-index-1;
@extend .pr-1;
@extend .text-secondary;
@extend .border-topy-3;
@extend .d-block;
@extend .h-100;
@extend .w-100;
@extend .position-relative;
@extend .z-index-1;
@extend .pr-1;
@extend .text-secondary;
@extend .border-top;
}

```

This way makes the html cleaner and easier to maintain the page